src/effects/native/reverbeffect.h

```diff
@@ -16,24 +16,19 @@
 #include "util/sample.h"
 #include "util/types.h"
 
+const CSAMPLE MIN_VALUE = 0.0005;
```

Comment:
Looks like these aren't used in the header. Could you put these in an anonymous namespace in reverbeffect.cpp and name them like 

```c++
namespace { 
const CSAMPLE kMinValue = ...;
}  // namespace
```